### DISCUSSION TITLE
Post-Judging QA

### DISCUSSION THREAD

**Discussion by JeeberC4** [code-423n4/2022-06-nested-findings#101] [Announcements]
## 🧑‍⚖️ Post-judging QA - Deadline July 15 09:00PM PST

1. Please post feedback about *specific findings* on the relevant issue in this repo.
2. More general feedback for the judge can be shared here. 

The judge for this contest is @dmvt. 

## Reminders

- Please keep your comments as fact-based as possible.
- C4 judges have final authority in determining validity and severity.
- You are welcome to start new discussions here!
- Please link to specific issues wherever relevant. 

Thank you!

**Comment by csanuragjain** [comment]
@jack-the-pug For below issues, I think funds can be placed at direct risk by owner for there own personal interest. Team mentioned that owner is trusted but I cannot see any problem in implementing a max fees which will eliminate this issue.

https://github.com/code-423n4/2022-06-nested-findings/issues/26
https://github.com/code-423n4/2022-06-nested-findings/issues/35
https://github.com/code-423n4/2022-06-nested-findings/issues/86

An example is if enough users have created NFT in contract and owner does not want users to leave the system. For this owner can set very high exit fees making destroy or withdraw process worthless for users. 

Can you please suggest if it make sense to elevate the severity to Medium for these issues

**Reply by jack-the-pug** [reply]
Per the doc (in the "Ownership & Governance" section):

> ...introduces the "Emergency Role" to execute a transaction in an instantaneous way. Only the "Emergency Multisig" has this role, with 5 members and 5 approvals needed (in the case of an urgent fix). On the other hand, the "Operational Multisig" can schedule/execute transactions with a 6-hours delay, with 3 members and 2 approvals needed.

1, the issue itself, is not very practical given that there is a timelock and the emergency role is a 5/5 multisig;
2, but I do agree that implementing upper bounds to the fees is better;
3, this is an easy issue; as auditors, we are supposed to identify the unknown issues: logic errors, errors in math, unexpected external behaviors, edge cases, etc; If the issue is likely to be known by the sponsor beforehand, I tend to lower the severity;
4, most of the issues didn't even mention timelock, and the one that mentioned it (#86) didn't read the doc; that's negligence.

For the reasons above, I'll keep the `QA` rating of these issues.

**Reply by jack-the-pug** [reply]
Thank you for pointing it out! @csanuragjain

**Reply by csanuragjain** [reply]
Makes sense thanks
